README.md

```diff
@@ -173,6 +171,40 @@ Parameters should be defined as follows:
 You can also run tests with `-v snipeit:no` in order to skip checking whether
 the platform is available on snipeit.
 
+## QEMU workflow
+
+Many of the test and keywords can be tested in emulation environment. This
+can greatly increase the development speed:
+* there is no need to acquire hardware,
+* there is no need to flash hardware, or resolve other hardware-related
+  problems,
+* the boot time (and responsivness in general) is much faster.
+
+Dasharo in QEMU can be started with:
+
+```bash
+./scripts/ci/qemu-run.sh
+```
+
+A graphical QEMU windows would popup, so you can observe the test flow, or
+control it manually. The actual testing will happen over serial, which is
+exposed via telnet.
+
+You may also build customized Dasharo firmware for QEMU (e.g. with some Dasharo
+options enabled or disabled). In such a case, please refer to:
+* [Building Manual in Dasharo for QEMU documentation](https://docs.dasharo.com/variants/qemu_q35/building-manual/)
+* [Development section in Dasharo for QEMU documentation](https://docs.dasharo.com/variants/qemu_q35/development/)
+
+Following tests are proven to work with QEMU right now:
+
+```shell
+robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no dasharo-compatibility/uefi-shell.robot
+robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no dasharo-compatibility/custom-boot-menu-key.robot
+```
```

Comment:
The assumption here is that someone has correctly configured the environment. Maybe a link would be needed to the above section if one would be linked directly here?

Reply:
2346889fbe5b87eb0a4d184cef4b0dc3fd6a035e